pkg/relayer/cmd/cmd.go

```diff
@@ -0,0 +1,250 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/url"
+	"os"
+	"os/signal"
+
+	"cosmossdk.io/depinject"
+	cosmosclient "github.com/cosmos/cosmos-sdk/client"
+	cosmosflags "github.com/cosmos/cosmos-sdk/client/flags"
+	cosmostx "github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+
+	"github.com/pokt-network/poktroll/pkg/client/block"
+	eventsquery "github.com/pokt-network/poktroll/pkg/client/events_query"
+	"github.com/pokt-network/poktroll/pkg/client/supplier"
+	"github.com/pokt-network/poktroll/pkg/client/tx"
+	"github.com/pokt-network/poktroll/pkg/relayer"
+	"github.com/pokt-network/poktroll/pkg/relayer/miner"
+	"github.com/pokt-network/poktroll/pkg/relayer/proxy"
+)
+
+var (
+	signingKeyName string
+	smtStorePath   string
+	sequencerNode  string
+	pocketNode     string
```

Comment:
Please look at `pkg/appgateserver/cmd/cmd.go` and the convention we added for local flag names.

```go
var (
	flagSigningKey        string
	flagSelfSigning       bool
	flagListeningEndpoint string
	flagCometWebsocketUrl string
)
```